pkg/types/command.go

```diff
@@ -0,0 +1,187 @@
+package types
+
+import (
+	"fmt"
+	"os/exec"
+)
+
+type Command struct {
+	// Print packets on stderr
+	Debug bool
+
+	// Length of packet
+	// Larger packets means less packets to exchange for the same amount of data (and less protocol overhead)
+	MTU int
+
+	// Values passed in by forward-xxx flags in commandline (forward-xxx:info)
+	forwardInfo map[string][]string
+
+	// List of endpoints the user wants to listen to
+	Endpoints []string
```

Comment:
Fwiw, having `AddEndpoint` and `endpointToCmdLine` helpers may be nicer than lettig the user deal directly with the `[]string`. This is similar to what you did with the maps.
Just a suggestion, I'm also fine with the code as it is now.